bundle/Templating/Twig/BetterIbexaAdminUIRuntime.php

```diff
@@ -4,26 +4,26 @@
 
 namespace Netgen\Bundle\BetterIbexaAdminUIBundle\Templating\Twig;
 
-use Ibexa\Contracts\Core\Repository\Values\Content\Query;
+use Ibexa\Contracts\Core\Repository\ContentService;
 use Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion;
-use Netgen\IbexaSiteApi\API\FilterService;
+use Ibexa\Contracts\Core\Repository\Values\Filter\Filter;
 use Twig\Extension\RuntimeExtensionInterface;
 
 final class BetterIbexaAdminUIRuntime implements RuntimeExtensionInterface
 {
-    private FilterService $filterService;
+    private ContentService $filterService;
```

Comment:
The property should also be renamed to `$contentService`

Reply:
That's embarrassing.